xDhcpServer.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.1'
+ModuleVersion = '1.1.1'
```

Comment:
Please, don't update module version https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#creating-a-new-dsc-resource-in-an-existing-module